includes/Checker/Checks/I18n_Usage_Check.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Class WordPress\Plugin_Check\Checker\Checks\I18n_Usage_Check
+ *
+ * @package plugin-check
+ */
+
+namespace WordPress\Plugin_Check\Checker\Checks;
+
+/**
+ * Check for running WordPress internationalization sniffs.
+ *
+ * @since n.e.x.t
+ */
+class I18n_Usage_Check extends Abstract_PHP_CodeSniffer_Check {
+
+	/**
+	 * List of I18n check arguments.
+	 *
+	 * @since n.e.x.t
+	 * @var array
+	 */
+	public $allowed_args = array(
+		'extensions' => 'php',
+		'standard'   => 'WordPress,WordPress-Core,WordPress-Docs,WordPress-Extra',
```

Comment:
@vishalkakadiya do we need to use all these standards? Will either `WordPress` or `WordPress-Core` work?

Reply:
@jjgrainger Used `WordPress` only, thank you!